Makefile

```diff
@@ -17,6 +17,8 @@ setaudit:	$(OBJS)
 install:	setaudit
 		mkdir -p $(PREFIX)/bin/
 		install -m 0555 setaudit $(PREFIX)/bin/setaudit
+		mkdir -p $(PREFIX)/man/man1/
```

Comment:
Do we need a: 
```shell
gzip -k setaudit.1
```
Here? It looks like everything in /usr/local/man/manX is compressed.

Reply:
From my experience the ports framework compresses man pages automagically behind the scenes. 

There's no need to do it, I think.